From c3b31b37b9dfb522f790cbcaeb7ed3216eeb8960 Mon Sep 17 00:00:00 2001
From: P. Christeas <p_christ@hol.gr>
Date: Thu, 28 Apr 2011 13:09:47 +0300
Subject: [PATCH 5/6] server-init: don't start OpenERP by default

According to Fedora packaging guidelines, we shouldn't have OpenERP start
by default on these systems. Admins must explicitly enable the service.

diff --git a/doc/openerp-server.init b/doc/openerp-server.init
index fdd8521..557a0e3 100644
--- a/doc/openerp-server.init
+++ b/doc/openerp-server.init
@@ -2,7 +2,7 @@
 # openerp-server        This shell script takes care of starting and stopping
 #                       OpenERP server
 #
-# chkconfig: 345 95 05
+# chkconfig: - 95 05
 # description: OpenERP server
 #
 # pidfile: /var/run/openerp-server.pid
@@ -12,8 +12,6 @@
 # Provides: openerp-server
 # Should-Start: $network $local_fs postgresql
 # Should-Stop: postgresql
-# Default-Start: 3 4 5
-# Default-Stop: 3 4 5
 # Short-Description: Launches the OpenERP server.
 # Description: This startup script launches the OpenERP server.
 ### END INIT INFO
-- 
1.7.4.4

